bokehjs/src/lib/models/widgets/abstract_slider.ts

```diff
@@ -33,6 +33,10 @@ abstract class AbstractBaseSliderView extends OrientedControlView {
 
   private _noUiSlider: API
 
+  get _steps(): API["steps"] {
```

Comment:
this seems a little awkward, @mattpap any other improvement possible?  This is my only comment about the PR otherwise

Reply:
Agreed. And just to clarify - I only put this in here so I could validate the slider step in tests. Alternative I considered was removing the private protection on `_noUiSlider`

Reply:
@g-parki does `API.steps` work here? That would be slightly preferable, if so.

Reply:
That's what I first tried. It errored in my editor:
` Cannot access 'API.steps' because 'API' is a type, but not a namespace. Did you mean to retrieve the type of the property 'steps' in 'API' with 'API["steps"]'?ts(2713)`